app/script/view_model/content/InputBarViewModel.js

```diff
@@ -361,7 +361,7 @@ z.viewModel.content.InputBarViewModel = class InputBarViewModel {
       this.sendMessage(messageText);
     }
 
-    this.currentMentions.removeAll();
+    this.currentMentions = [];
```

Comment:
Let's not always create a new array. You can do `this.currentMentions.length = 0`